scripts/variantstore/beta_docs/gvs-workspace-description.md

```diff
@@ -0,0 +1,128 @@
+### Genomic Variant Store workflow (beta) for variant discovery
+
+The [Genomic Variant Store (GVS)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/gvs-product-sheet.pdf) was developed as a solution for variant discovery on a large scale. The GVS is powered by BigQuery and creates large joint callsets more reliably with decreased time and cost compared to previous solutions.
+
+This workspace contains a fully reproducible example workflow for variant discovery using the GVS workflow.
+
+Scroll down for an overview of the workflow, example data, cost estimates, and additional resources.
+
+The materials in this workspace were developed by the Data Sciences Platform at the Broad Institute.
+
+## Workflow Overview
+
+![A diagram depicting the Genomic Variant Store workflow. Sample GVCF files are imported into the core data model. A filtering model is trained using Variant Quality Score Recalibration, or VQSR, and then applied while the samples are extracted as cohorts in sharded joint VCF files. Each step integrates BigQuery and GATK tools.](https://storage.googleapis.com/terra-featured-workspaces/Genomic-Variant-Store/genomic-variant-store_diagram.png)
+
+### What does it do?
+
+The [GVS workflow](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) is an open-source, cloud-optimized workflow for joint calling at a large scale using the GVS. The workflow takes in single sample GVCF files, loads them into [BigQuery](https://cloud.google.com/bigquery/docs) tables, and combines them into a variant filtering model driven by machine learning. The model is uploaded back into BigQuery and applied to the data. The workflow produces sharded joint VCF files with indices, a manifest file, and metrics.
+
+For workflow documentation, see the [Genomic Variant Store workflow overview](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/beta_docs/gvs-overview.md).
+
+### What data does it require as input?
+
+- Reblocked single sample GVCF files (`input_vcfs`)
+- GVCF index files (`input_vcf_indexes`)
+
+To see details about input requirements, see [Run Your Own Samples](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/beta_docs/run-your-own-samples.md). Example GVCF and index files in the Data tab of this workspace are hosted in a public Google bucket and links are provided in the sample data table.
+
+While the GVS has been tested with 250,000 single sample genome GVCF files as input, only datasets of up to 25,000 genomes are being used for beta testing.
+
+### What does it return as output?
+
+The following files are stored in the workspace workflow execution bucket under Data>Files, or the google bucket specified in the inputs.
```

Comment:
if you think my more wordy version of "Data>Files" above is better, maybe a good idea to include here, too?